### PR TITLE
Update readme.md - suggestion for a dinner recipe

### PR DESCRIPTION
**Pull request by amypaddle1**
## 🎯 Objective
Proposing an easy pasta Alfredo recipe to get something started

Related to issue: https://github.com/IntroToCode/CodeMates7_Fall22/issues/2#issue-1420006599

## 🛠️ Changes made

added recipe name, servings, time, ingredients, steps

## 👥 Who needs to review this Pull Request? 
@p-mullangi
@carolinadarraidou
@gurbuxlohana
@rnagpal991

### CODE REVIEW

**Review by amypaddle1 — COMMENTED** [review]
@p-mullangi
@carolinadarraidou
@gurbuxlohana
@rnagpal991

What do you think?

**Review by carolinadarraidou — COMMENTED** [review]
I would add extra parmesan cheese a gusto in the end

**Review by carolinadarraidou — APPROVED** [review]

File: dinner/readme.md [hunk 1]
```diff
@@ -1,21 +1,33 @@
...
-
+1. Cook the pasta according to directions on the package, then drain and set aside. 
+2. Add stick of butter along with 1 cup heavy cream to a large skillet over medium heat. Allow butter to melt.
+3. Simmer for 5 minutes on low heat.
+4. Add 1 tbs garlic powder, 1/2 tsp salt, 1/2 tsp black pepper, and 2 cups of parmesan cheese to the skillet and whisk until cheese has melted.
+5. Serve the sauce over cooked pasta and top with 2tbs of chopped parsley.
 
 
 ### Notes:
+This sauce pairs well with grilled chicken, shrimp, and veggies too!
```
Comment (carolinadarraidou) [inline dinner/readme.md:31]: Add extra parmesan cheese a gusto

**Comment by amypaddle1** [comment]
@aparnachaganty @joewadcan 

Hi, I'm still confused by who has write access and merge a pull request. I know we need two reviewers for the purpose of the assignment but I was still expecting to have the ability to merge. What am I missing? 👀 😕

**Review by p-mullangi — APPROVED** [review]
LGTM (looks good to me)

**Pull request merged**